src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Unix.cs

```diff
@@ -113,6 +114,17 @@ private static bool IsDistroAndVersion(string distroId, int major = -1, int mino
             return IsDistroAndVersion((distro) => distro == distroId, major, minor, build, revision);
         }
 
+        /// <summary>
+        /// Get whether the OS platform matches the given Linux distro and optional version is same or higher.
+        /// </summary>
+        /// <param name="distroId">The distribution id.</param>
+        /// <param name="versionId">The distro version.  If omitted, compares the distro only.</param>
+        /// <returns>Whether the OS platform matches the given Linux distro and optional version is same or higher.</returns>
+        private static bool IsDistroAndVersionOrHigher(string distroId, int major = -1, int minor = -1, int build = -1, int revision = -1)
+        {
+            return IsDistroAndVersionOrHigher((distro) => distro == distroId, major, minor, build, revision);
```

Comment:
Nit I don't think you need parens around distro. i would write
`return IsDistroAndVersionOrHigher(distro => (distro == distroId), major, minor, build, revision);`